frontend/src/components/user/MyProfile.js

```diff
@@ -99,7 +137,11 @@ const MyProfile = ({ user }) => {
       if (password.value) data.password = password.value;
       if (confirmPassword.value) data.confirmPassword = confirmPassword.value;
       if (lang) data.language = lang.code;
+      if (userData) data.avatarBase64String = avatar;
```

Comment:
It looks like when we update user information, the avatar (avatarBase64String) always sent to the backend, even if we didn't change it. According to our documentation, we only update information if the change has been done to it. Is it possible to do that? Of course, we can keep it working like that if changes require a lot of work.
By the way, great work on profile :)